### ISSUE TITLE
Meetup Suggestion: *Bitcoin Lausanne, Switzerland

### ISSUE THREAD

**Issue by sven-hash** [bitcoin-only/bitcoin-only#684]
Thanks for making a suggestion for a new meetup.

Please note: Only suggest bitcoin-only meetups. 
Meetups which promote any coin or token other than Bitcoin are not suitable for listing. 
Altcoin's are only mentioned in the context of warnings or lessons learnt **for bitcoin**.

## Required:
1. Name of Meetup: Bitcoin Lausanne
2.  Location Of Meetup: Lausanne, Switzerland
3. At least one of the following:
- Meetup Page (or similar): https://www.meetup.com/bitcoin-lausanne/
- Telegram Group (or similar): https://t.me/BitcoinLausanne
- Website: https://bitcoinlausanne.ch/

## Optional 
- Name of Organiser(s) & Twitter Handle: Albin CAUDERLIER / https://twitter.com/AlbinCAUDERLIER
- Links showing public support: >>>  https://twitter.com/cg1_bin/status/1572539303275950081?s=20&t=TotjrIXYoMmMDh2TkdkZ8w
- Tweet your commitment 

```
As the organiser of Bitcoin Lausanne Meetup I can confirm that the meetup is bitcoin-only
-The meetup is bitcoin focused.
-'Altcoins' are scams and are not promoted in talks/presentations at our meetups.
#bitcoinonlymeetups @bitcoinonly_
```

**Comment by 6102bitcoin** [comment]
Founder of the meetup runs https://mubiz.com/ which supports Ethereum. Are you sure this is bitcoin-only?

**Comment by sven-hash** [comment]
Yes the founder have an company where he accept ETH but the meetup is not related to mubiz
